config/default-config.json

```diff
@@ -27,6 +27,7 @@
     "default theme": "oc",
     "themes supported": [],
     "base path": "",
+    "home directory": "/usr/local/git/enketo-express-oc",
```

Comment:
seems silly to add its own location as a config item

Reply:
could you suggest a better way to do this? I can't think of anything.

Reply:
Unfortunately, I don't quite know how to reproduce the problem, so I cannot test myself. But I was thinking of these (possibly with steps down and up using [path.join()](https://nodejs.org/docs/latest-v12.x/api/path.html#path_path_join_paths)):

* `process.cwd()`
* `__dirname`

https://stackoverflow.com/questions/9874382/whats-the-difference-between-process-cwd-vs-dirname

Reply:
I tested it and the __dirname works. Please code review new code, thank you!

Reply:
That's great. Nice and simple. I've also added it to enketo/enketo-express.